build/index.js

```diff
@@ -262,7 +262,7 @@ async function main() {
 
     const eslintOptions = ["--ext", ".ts,.js,.tsx,.jsx", "."];
 
-    if (command === "build" || command === "lint") {
+    if (command !== "list" && command !== "list-deps" && command !== "clean") {
```

Comment:
Nit: I'd find it a bit cleaner like this: `if (!["list", "list-deps", "clean"].includes(command)) {` - but doesn't really matter :D

Reply:
Cleaner is always better